SEP/SEP-0003/sep-0003.md

```diff
@@ -0,0 +1,44 @@
+# Property paths with a min/max hop
+
+* Short Name: min-max-propery-path
+* SEP Number: [SEP-0003](sep-0003.md)
+* Author: [Jerven Bolleman](https://github.com/jervenbolleman)
+
+## Abstract
+
+In the SPARQL 1.1. working group, a form of bounded property path where supported in the late [drafts](https://www.w3.org/TR/2012/WD-sparql11-query-20120105/#propertypaths). The form being a path element _elt_, followed by a curly bracket enclosed min/max pattern.
+
+|Syntax Form|Matches|
+|-----------|-----------|
+| _elt_{n,m} | A path of between n and m occurrences of elt. |
```

Comment:
To discuss? must `m` and `n` be a constant or are variables allowed?

Reply:
Supporting variables would require a major change to the evaluation semantics.